src/features/toolbar/components/ToolbarControl/ToolbarControl.tsx

```
@@ -49,6 +49,10 @@ export function ToolbarControl({
       const size = isMobile
         ? (settings.typeSettings.mobilePreferredSize ?? 'medium')
         : settings.typeSettings.preferredSize;
+      const hint = resolveValue(
+        settings.typeSettings.hint ?? settings.typeSettings.name,
+        state,
+      );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)